doc/authentication-for-emergency-department/index.md

```diff
@@ -1,34 +1,37 @@
 # Authentication for emergency department
 
-
 **Status**: Experiment by IGDC-DHCW + draft + work in progress + request for comments
 
-**Date**: 2025-03-24
+**Date**: Updated 2025-04-07
 
 **Governance**: IGDC-DHCW for the time being
 
-
 ## Situation - Context and Problem Statement
 
-We are developing a hospital emergency department software application, and we are learning that some kinds of authentication are not viable e.g. sending the doctor an SMS text, or using a TOPT authenticator app on a nurse's mobile phone.
+We are developing a hospital emergency department software application, and we
+are learning that some kinds of authentication are not viable e.g. sending the
+doctor an SMS text, or using a TOPT authenticator app on a nurse's mobile phone.
 
 This is a work in progress (WIP) and a request for comments (RFC).
 
-We welcome constructive advice, constructive criticism, pointers to resources, and help.
+We welcome constructive advice, constructive criticism, pointers to resources,
+and help.
 
 Direct contact information:<br>
 Joel Henderson <a href="mailto:joel.henderson@wales.nhs.uk">joel.henderson@wales.nhs.uk</a>
```

Comment:
Need to remove HTML tags (lint rule) - can wrap this in Markdown: ``[joel.henderson@wales.nhs.uk](mailto:joel.henderson@wales.nhs.uk)``